src/main/java/com/openlattice/chronicle/services/download/DataDownloadService.kt

```diff
@@ -59,64 +70,66 @@ class DataDownloadService(private val entitySetIdsManager: EntitySetIdsManager,
-        val srcPropertiesToExclude: Set<FullQualifiedName> = ImmutableSet.of(EdmConstants.STRING_ID_FQN, com.openlattice.edm.EdmConstants.ID_FQN)
-        val edgePropertiesToExclude: Set<FullQualifiedName> = ImmutableSet.of(EdmConstants.STRING_ID_FQN, EdmConstants.DATE_LOGGED_FQN, com.openlattice.edm.EdmConstants.ID_FQN)
+            studyId: UUID,
+            participantEntityId: UUID,
+            dataType: ParticipantDataType,
+            token: String?
+    ): Iterable<Map<String, Set<Any>>> {
 
         return getParticipantDataHelper(
+                organizationId,
+                studyId,
                 participantEntityId,
-                entitySetIdGraph,
-                srcPropertiesToExclude,
-                edgePropertiesToExclude,
+                dataType,
                 token
         )
     }
 
-    override fun getAllParticipantData(
-            organizationId: UUID?,
-            studyId: UUID?,
-            participantEntityId: UUID?,
-            token: String?): Iterable<Map<String, Set<Any>>> {
-        val entitySetIdGraph = getEntitySetIdGraph(organizationId, studyId, ParticipantDataType.RAW_DATA)
-        val srcPropertiesToExclude: Set<FullQualifiedName> = ImmutableSet.of(EdmConstants.STRING_ID_FQN, com.openlattice.edm.EdmConstants.ID_FQN)
-        val edgePropertiesToExclude: Set<FullQualifiedName> = ImmutableSet.of(EdmConstants.STRING_ID_FQN, EdmConstants.DATE_LOGGED_FQN, com.openlattice.edm.EdmConstants.ID_FQN)
-
-        return getParticipantDataHelper(
-                participantEntityId,
-                entitySetIdGraph,
-                srcPropertiesToExclude,
-                edgePropertiesToExclude,
-                token
-        )
+    private fun getColumnTitles(propertyTypes: Set<FullQualifiedName>, metadata: Map<UUID, EntitySetPropertyMetadata>, prefix: String): List<String> {
+        return propertyTypes.map {
+            val propertyTypeId = edmCacheManager.getPropertyTypeId(it)
+            prefix + metadata.getValue(propertyTypeId).title
+        }
     }
 
-    override fun getAllParticipantAppsUsageData(
-            organizationId: UUID?,
-            studyId: UUID?,
-            participantEntityId: UUID?,
-            token: String?): Iterable<Map<String, Set<Any>>> {
-        val entitySetIdGraph = getEntitySetIdGraph(organizationId, studyId, ParticipantDataType.USAGE_DATA)
-        val srcPropertiesToExclude: Set<FullQualifiedName> = ImmutableSet.of(EdmConstants.STRING_ID_FQN, com.openlattice.edm.EdmConstants.ID_FQN)
-        val edgePropertiesToExclude: Set<FullQualifiedName> = ImmutableSet.of(EdmConstants.STRING_ID_FQN, com.openlattice.edm.EdmConstants.ID_FQN)
+    private fun getSrcPropertiesToInclude(dataType: ParticipantDataType): Set<FullQualifiedName> {
+        return when (dataType) {
+            ParticipantDataType.USAGE_DATA -> linkedSetOf(TITLE_FQN, FULL_NAME_FQN)
```

Comment:
this is great! if you wanted to take it one step further you could save this as a static map from ParticipantDataType to Set<FQN> to avoid unnecessary construction + computation (even though the computation is negligible, this is effectively supposed to be a constant so it's logically a bit cleaner to store it as such).

If you wanted to do that, you could do it something like:
```
class DownloadTypePropertyTypeFqns {
  companion object {

    val SRC = mapOf(
      ParticipantDataType.USAGE_DATA to linkedSetOf( ... ),
      ParticipantDataType.RAW_DATA to linkedSetOf( ... ),
      ParticipantDataType.PREPROCESSED to linkedSetOf( ... )
    )

    val EDGE = mapOf(
      ParticipantDataType.USAGE_DATA to linkedSetOf( ... ),
      ParticipantDataType.RAW_DATA to linkedSetOf( ... ),
      ParticipantDataType.PREPROCESSED to linkedSetOf( ... )
    )

  }
}
```

(and then instead of calling this function, you'd just do `val srcPropertiesToInclude = DownloadTypePropertyTypeFqns.SRC.getValue( datatype )`

Again, just a minor organizational thing, and not important for this PR!